share/man/man7/arch.7

```diff
@@ -335,10 +367,11 @@ cc -x c -dM -E /dev/null
 .Ed
 .Pp
 Common type size and endianness macros:
-.Bl -column -offset indent "BYTE_ORDER" "Meaning"
+.Bl -column -offset indent "__CHERI_PURE_CAPABILITY__" "Meaning"
 .It Sy Macro Ta Sy Meaning
 .It Dv __LP64__ Ta 64-bit (8-byte) long and pointer, 32-bit (4-byte) int
 .It Dv __ILP32__ Ta 32-bit (4-byte) int, long and pointer
+.It Dv __CHERI_PURE_CAPABILITY__ Ta 128-bit (16-byte) capability and pointer, 64-bit (8-byte) long
```

Comment:
This is so long.  Maybe it should just say that pointers are the same size as capabilities?  But FreeBSD doesn't support any 32-bit CHERI arches and isn't likely to in the future.

Reply:
I meant "capability pointer" to distinguish from 128-bit architectures